tests/app/test_agenting.py

```diff
@@ -245,19 +272,189 @@ def test_witnesser(helpers):
         doist.recur(deeds)
 
 
+def test_submitter(seeder, helpers):
```

Comment:
Test the submitter by creating a test agent, a test witness hab, and a separate witness service. This allows us to partially witness and then 'submit' in order to completely witness.